src/ledmatrixgfx.cpp

```diff
@@ -88,7 +88,7 @@ void LEDMatrixGFX::PrepareFrame()
             backgroundLayer.drawString(MATRIX_WIDTH / 2 - (3 * output.length()), MATRIX_HEIGHT / 2 - 5, rgb24(255, 255, 255), rgb24(0, 0, 0), output.c_str());
         #endif
 
-        auto graphics = g_ptrSystem->EffectManager()[0];

```

Comment:
It's equivalent, but I'm not a fan of operator overloading, and it hid a bug, so changed to the g() function which was there anyway.

Reply:
That's a good clean-up, and increases consistency with how the first graphics channel is retrieved everywhere else already. I'm now wondering why that didn't click when I looked at this before.